client/context/helpers.go

```diff
@@ -345,3 +348,28 @@ func (ctx CoreContext) GetNode() (rpcclient.Client, error) {
 	}
 	return ctx.Client, nil
 }
+
+func getReadableString(a interface{}) {
```

Comment:
Also, do you think there is a more appropriate place for this as it doesn't have a solid relationship with a context?

Reply:
I think so. One of the purposes of this pr is that I want to know where more appropriate place is . 😅

Reply:
I think so. One of the purposes of this pr is that I want to know where the more appropriate place is . 😅